src/libstd/net/udp.rs

```diff
@@ -303,15 +312,36 @@ impl UdpSocket {
     /// Connects this UDP socket to a remote address, allowing the `send` and
     /// `recv` syscalls to be used to send data and also applies filters to only
     /// receive data from the specified address.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::net::UdpSocket;
+    ///
+    /// let socket = UdpSocket::bind("127.0.0.1:34254").expect("couldn't bind to address");
+    /// socket.connect("127.0.0.1:8080").expect("couldn't connect to server");
```

Comment:
No, you do not connect to *anything* with UDP. Server will never know that you… uh… “connect” to it. The only thing that really changes is some internal field called `default_send_address`.

`connect` method on `UdpSocket` is only named this way because its named this way on the UNIX side, but you oughtn’t be misled by the naming.